app/frontend/wallet/shelley/shelley-transaction-planner.ts

```diff
@@ -121,7 +141,11 @@ export function estimateTxSize(
 
   const txWitnessesSize = shelleyWitnessesSize + byronWitnessesSize
 
-  const txMetaSize = 1 // currently null
+  const placeholderMetaSignature = 'x'.repeat(CATALYST_SIGNATURE_BYTE_LENGTH * 2)
+  const txAuxiliaraDataSize = auxiliaryData
```

Comment:
I'd expect here some switch statement which would apply only if the auxiliary data passed is a voting registration, so typescript warns us if we happen to serialize other kind of metadata, where this formula wouldn't work